monitor.go

```diff
@@ -151,6 +165,34 @@ func newDockerClient() (*docker.Client, error) {
 	return docker.NewTLSClient(dockerHost, cert, key, ca)
 }
 
+func NewPodmanClient() (*podmanClient, error) {
```

Comment:
You could probably define a simple interface to abstract away the new Podman and old Docker client.
That way, you don't have to have separate codes for each client in your main business logic.

Reply:
I would start by listing the current set of operations that are being used from the existing docker client, and see if the same thing can be done in the Podman lib. Then you could define a Go interface that abstract both the Docker and your new Podman implementation.